packages/web-app-external/src/App.vue

```diff
@@ -1,27 +1,27 @@
 <template>
   <main
-    class="uk-height-viewport"
+    class="full-height"
```

Comment:
Could you use the helper-class `uk-height-1-1` instead of defining your custom CSS? :)

Reply:
@elizavetaRa